code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -1,5 +1,5 @@
 /mob/living/carbon/human
-	hud_possible = list(HEALTH_HUD,STATUS_HUD,ID_HUD,WANTED_HUD,IMPLOYAL_HUD,IMPCHEM_HUD,IMPTRACK_HUD, NANITE_HUD, DIAG_NANITE_FULL_HUD,ANTAG_HUD,GLAND_HUD,SENTIENT_DISEASE_HUD,RAD_HUD)
+	hud_possible = list(HEALTH_HUD,STATUS_HUD,ID_HUD,WANTED_HUD,IMPLOYAL_HUD,IMPCHEM_HUD,IMPTRACK_HUD, NANITE_HUD, DIAG_NANITE_FULL_HUD,ANTAG_HUD,GLAND_HUD,SENTIENT_DISEASE_HUD,RAD_HUD,ANTAGTARGET_HUD)
```

Comment:
I'm not sure if this can be modularized by adding to the list on Initialize() but it'd be worth it to try

Reply:
My BYOND skills are not good enough to know how to do that : <

Reply:
Basically just go to a modular file and throw something like this in

```
/mob/living/carbon/human/Initialize()
	LAZYADD(hud_possible, ANTAGTARGET_HUD)
	. = ..()
```